eng/versioning/external_dependencies.txt

```diff
@@ -160,7 +160,7 @@ org.springframework:spring-jdbc;5.3.24
 org.springframework:spring-jms;5.3.24
 org.springframework:spring-messaging;5.3.24
 org.springframework:spring-tx;5.3.24
-org.springframework:spring-web;5.3.24
+org.springframework:spring-web;6.0.0
```

Comment:
We'll want to wait for the Spring team to make this change as `6.0.0` and later begins requiring Java 17 as the baseline version.